centrapay/centrapay-docs
# Add missing description and image metadata for the home page

Add a description and image metadata for the home page so we provide more information for search engines and render nice link previews.

Center the icon in our support button as a quick fix.

Test plan:
- Confirm that the description and image are present in the `<head>` of the homepage
- Confirm that the description and image are rendered in link previews

![image](https://user-images.githubusercontent.com/46909693/231899149-21f484ef-6674-4bde-9efe-d60f50c84d6c.png)
